management/virtualmachine/client.go

```diff
@@ -665,7 +665,7 @@ func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH
 	publicKey.Fingerprint = fingerprint
 	publicKey.Path = "/home/" + userName + "/.ssh/authorized_keys"
```

Comment:
This is the path where the authorized_keys file is on the vm being deployed. Not the certificate. That should already exist in the cloud service...

Sent from my Windows Phone
________________________________
From: Ahmet Alp Balkan
Sent: 3/14/2015 2:27 PM
To: MSOpenTech/azure-sdk-for-go
Cc: Paul Meyer
Subject: Re: [azure-sdk-for-go] Update virtual machine entities to latest spec (#64)


In management/virtualmachine/client.go<https://github.com/MSOpenTech/azure-sdk-for-go/pull/64#discussion_r26441925>:

> @@ -665,7 +665,7 @@ func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH
>       publicKey.Fingerprint = fingerprint
>       publicKey.Path = "/home/" + userName + "/.ssh/authorized_keys"


do you think we can just accept certPath as input to this method? hardcoded home path wouldn't work on windows or mac.

-
Reply to this email directly or view it on GitHub<https://github.com/MSOpenTech/azure-sdk-for-go/pull/64/files#r26441925>.